### PR TITLE
fix: npm scripts not working on windows

### LINKED ISSUE #53: Windows: npm scripts not working
Cannot start npm scripts on windows platform: it is using `cp` linux command.

I'll make a PR in a few minutes.

### PR DESCRIPTION
**Pull request by userquin**
closes #53

Also includes `.idea/` directory for `intellij/webstorm` on `.gitignore` file.

### CODE REVIEW

**Comment by antfu** [comment]
Emm, we can wrote a simple javascript to copy the files instead of using cli.

File: package.json [hunk 1]
```diff
@@ -44,21 +44,16 @@
   ],
   "scripts": {
     "dev": "npm run build -- --watch",
-    "example:dev": "cp README.md examples/multiple-pages/README.md && npm -C examples/multiple-pages run dev",
-    "example:build": "cp README.md examples/multiple-pages/README.md && npm -C examples/multiple-pages run build",
-    "example:serve": "cp README.md examples/multiple-pages/README.md && npm -C examples/multiple-pages run serve",
-    "example:store:dev": "cp README.md examples/multiple-pages-with-store/README.md && npm -C examples/multiple-pages-with-store run dev",
-    "example:store:build": "cp README.md examples/multiple-pages-with-store/README.md && npm -C examples/multiple-pages-with-store run build",
-    "example:store:serve": "cp README.md examples/multiple-pages-with-store/README.md && npm -C examples/multiple-pages-with-store run serve",
+    "copy-files": "node scripts/copy-files.ts",
```
Comment (antfu) [inline package.json:47]:
```suggestion
    "copy-files": "esno scripts/copy-files.ts",
```

File: scripts/copy-files.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const fs = require('fs')
```
Comment (antfu) [inline scripts/copy-files.ts:2]: Using `esno` we could use `import` here

Comment (userquin) [reply scripts/copy-files.ts:2]: we need to add `esno` to `dev dependencies`, you have it globally...

Comment (userquin) [reply scripts/copy-files.ts:2]:
```shell
F:\work\projects\quini\GitHub\antfu\vite-ssg>pnpm run example:dev

> vite-ssg@0.11.2 example:dev F:\work\projects\quini\GitHub\antfu\vite-ssg
> npm run copy-files && npm -C examples/multiple-pages run dev


> vite-ssg@0.11.2 copy-files
> esno scripts/copy-files.ts

"esno" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.
 ERROR  Command failed with exit code 1.
```

**Pull request merged**